src/objectTranslations/Address__c-nl_NL.objectTranslation.orig

```diff
@@ -0,0 +1,1096 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@dmkud I can strip these out of the PR, but I thought you should know that they were included in the last commit.

Reply:
Noted.